ruby_on_rails-postgres/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.7'
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: 'username'
+      POSTGRES_HOST_AUTH_METHOD: trust
+  web:
+    build: ./web
+    command: bash -c "rm -f tmp/pids/server.pid && rake db:drop db:create db:migrate && bundle exec rails s -b 0.0.0.0"
```

Comment:
Why dropping the database each time we run the application? 
Maybe we should create a specific task to check if the database exist or not, and only create the db when needed?
Something like `command: bash -c "rm -f tmp/pids/server.pid && (rake db:exists && rake db:migrate || rake db:setup) && bundle exec rails s -b 0.0.0.0"`
And add a `lib/tasks/db_exists.rake` file with something like this
```
namespace :db do
  desc "Checks to see if the database exists"
  task :exists do
    begin
      Rake::Task['environment'].invoke
      ActiveRecord::Base.connection
    rescue
      exit 1
    else
      exit 0
    end
  end
end
```

Reply:
@glours since it has a `schema.rb` file, `rake db:reset` would be functionally equivalent to  `rake db:drop db:create db:migrate` commands. There'd no error even there is no database, no need to add additional task.